src/locales/en.json

```diff
@@ -52,66 +52,64 @@
         }
       },
       "beforeWeGetStarted": {
-        "p1": "Congratulations on opening this page! This means you will learn everything about CKB ahead of the rest if you are a beginner.",
-        "p2": "This tutorial is your first contact with CKB, either for developing a DAPP on CKB or simply understanding the basics out of curiosity. Follow this guide to get your first hands-on CKB experience.",
+        "p1": "Welcome to this page! From here, you will learn CKB ahead of everyone else.",
+        "p2": "In this tutorial, you will learn the basics of CKB, so that you can develop an application on CKB or simply understand them for curiosity's sake. Follow this guide to get your first hands-on experience with CKB.",
         "p3": "This tutorial is a complete walk-through that allows you to accomplish the following tasks by yourself:",
         "ul1": {
-          "l1": "Build and send the simplest transfer transaction",
-          "l2": "Develop and send the simplest multi-signature transaction",
-          "l3": "Develop and deploy the simplest smart contract",
-          "l4": "Develop and deploy a scalable smart contract"
+          "l1": "Build and send the simplest transfer transaction.",
+          "l2": "Develop and send the simplest multi-signature transaction.",
+          "l3": "Develop and deploy the simplest smart contract.",
+          "l4": "Develop and deploy a scalable smart contract."
         },
-        "p4": "The greatest thing is, to figure this all out, there is:",
+        "p4": "On this page you can interact directly with the cloud-based test chain in a manual, remote manner. All you need to have is the patience for reading further down and the curiosity for CKB.",
+        "p5": "To figure it all out, there is:",
         "ul2": {
           "l1": "No need for any local downloads",
           "l2": "No need to run any software",
           "l3": "No need even to write one single line of code!"
-        },
-        "p5": "Everything can be done on this page interactively with the cloud-based test chain in a purely manual, remote way.",
-        "p6": "All you need to have is the patience for reading further down and the curiosity for CKB. :D",
```

Comment:
can we make the line number not changed with Chinese version? due to the impl of front-end, since the Chinese version has `p6` and `p7` https://github.com/RetricSu/zero2ckb-web/blob/develop/src/locales/zh.json#L71-L72 the frontend will try rendering p6/p7 content, while in english version it will not find corresponding content.